libcxx/include/__thread/support/pthread.h

```diff
@@ -30,7 +30,9 @@
 // so libc++'s <math.h> usually absorbs atomic_wide_counter.h into the
 // module with <math.h> and makes atomic_wide_counter.h invisible.
 // Include <math.h> here to work around that.
-#include <math.h>
+#if __building_module(std)
```

Comment:
Please add comment about the module here too. `__building_module(std)` gives very strong vibes of being about the C++23 std module.